gradle.properties

```diff
@@ -3,7 +3,7 @@ defaultVersion=0.0.1-SNAPSHOT
 projectGroup=org.eclipse.dataspaceconnector.registrationservice
 
 edcGroup=org.eclipse.dataspaceconnector
-edcVersion=0.0.1-SNAPSHOT
+edcVersion=0.0.1-milestone-5
```

Comment:
we need to update checkout as well during the build ?
https://github.com/eclipse-dataspaceconnector/RegistrationService/blob/0b39c0b0cb1884af58d829fa8396cfa3cf609117/.github/actions/gradle-setup/action.yml#L12

Reply:
updated